### DISCUSSION TITLE
Version 2023-08-07 is not being built, errors are being issued

### DISCUSSION THREAD

**Discussion by caslix** [endink/Mediapipe4u-plugin#94] [Q&A]
Hello!
Version 2023-08-07 for UE5.1 generates the following error during assembly:

2>Building MediaPipe4U_DemoEditor...
2>Using Visual Studio 2022 14.37.32822 toolchain (C:\Program Files\Microsoft Visual Studio\2022\Community\VC\Tools\MSVC\14.37.32822) and Windows 10.0.22621.0 SDK (C:\Program Files (x86)\Windows Kits\10).
2>Determining max actions to execute in parallel (12 physical cores, 24 logical cores)
2>  Executing up to 12 processes, one per physical core
2>Building 2 actions with 2 processes...
2>[1/2] Link UnrealEditor-MediaPipeLLMSpeech.dll
2>   Создается библиотека C:\MediaPipe4U-Demo-main_20230617\Plugins\MediaPipe4ULLMSpeech\Intermediate\Build\Win64\UnrealEditor\Development\MediaPipeLLMSpeech\UnrealEditor-MediaPipeLLMSpeech.suppressed.lib и объект C:\MediaPipe4U-Demo-main_20230617\Plugins\MediaPipe4ULLMSpeech\Intermediate\Build\Win64\UnrealEditor\Development\MediaPipeLLMSpeech\UnrealEditor-MediaPipeLLMSpeech.suppressed.exp
2>LLMSpeechBlueprintLibrary.gen.cpp.obj : error LNK2019: ссылка на неразрешенный внешний символ "public: static void __cdecl ULLMSpeechBlueprintLibrary::SetSpeechActorForLLM(class AMediaPipeSpeechActor *)" (?SetSpeechActorForLLM@ULLMSpeechBlueprintLibrary@@SAXPEAVAMediaPipeSpeechActor@@@Z) в функции "public: static void __cdecl ULLMSpeechBlueprintLibrary::execSetSpeechActorForLLM(class UObject *,struct FFrame &,void * const)" (?execSetSpeechActorForLLM@ULLMSpeechBlueprintLibrary@@SAXPEAVUObject@@AEAUFFrame@@QEAX@Z).
2>C:\MediaPipe4U-Demo-main_20230617\Plugins\MediaPipe4ULLMSpeech\Binaries\Win64\UnrealEditor-MediaPipeLLMSpeech.dll : fatal error LNK1120: неразрешенных внешних элементов: 1
2>[2/2] WriteMetadata MediaPipe4U_DemoEditor.target cancelled
2>C:\Program Files\Microsoft Visual Studio\2022\Community\MSBuild\Microsoft\VC\v170\Microsoft.MakeFile.Targets(44,5): error MSB3073: выход из команды ""C:\Program Files\Epic Games\UE_5.1\Engine\Build\BatchFiles\Build.bat" MediaPipe4U_DemoEditor Win64 Development -Project="C:\MediaPipe4U-Demo-main_20230617\MediaPipe4U-Demo.uproject" -WaitMutex -FromMsBuild" с кодом 6.
2>Сборка проекта "MediaPipe4U-Demo.vcxproj" завершена с ошибкой.
========== Сборка: успешно выполнено — 0 , со сбоем — 1, в актуальном состоянии — 0, пропущено — 1 ==========
========== Сборка запущено в 17:42, заняло 03,965 с ==========

**Comment by endink** [comment]
Sorry, the demo project hasn't had time to be updated yet. 

You need clean your project, It looks like your project caches an old DLL file, and you should clean up the project.

BTW, please post issue at [issues](https://github.com/endink/Mediapipe4u-plugin/issues).

Discussion may be missed by me.

**Comment by endink** [comment]
I move this to issues:

https://github.com/endink/Mediapipe4u-plugin/issues/95

**Comment by endink** [comment]
fix at 20230811
